### PR TITLE
Fix - Host file location for windows os platform

### LINKED ISSUE #2: Got error on Windows OS
Windows Users don't have host file in `/etc/hosts` xD
capture

### PR DESCRIPTION
**Pull request by shivapoudel**
Fixes #2

### CODE REVIEW

**Comment by shivapoudel** [comment]
@fdaciuk don't know what happen to the coveralls, as it's showing failed test here :worried:

**Comment by shivapoudel** [comment]
@fdaciuk don't know what happen to the coveralls, as it's showing failed test here :worried:

File: lib/hoost-edit.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ function editHost( ip, host, newip, newhost ) {
 
 function readFile( err, data ) {
   if( err ) throw err;
+  fs.chmodSync( file, '0644' );
```
Comment (fdaciuk) [inline lib/hoost-edit.js]: -1
Why change the file permissions?

**Comment by fdaciuk** [comment]
> @fdaciuk don't know what happen to the coveralls, as it's showing failed test here

Don't worry about that :)
I will to fix it. We need more tests xD

File: lib/hoost-edit.js (hunk 1 above)
Comment (fdaciuk) [inline lib/hoost-edit.js]: I think this is a bad practice. For changing a file, the user needs to have permission to do this.
To editing, use `sudo`.

**Comment by fdaciuk** [comment]
Now I saw: the coverage fell because you changed the permissions but don't add more tests xD

**Comment by shivapoudel** [comment]
@fdaciuk In windows there is no `sudo` command so we have to set permission manually. I have remove the file permissions :)

**Comment by fdaciuk** [comment]
Hey man! I'm sorry for the late reply!
Thanks so much! :dancer: :dancer: :sparkles:

**Comment by shivapoudel** [comment]
Cool! can you update the version :P

**Comment by fdaciuk** [comment]
I did!

**Comment by shivapoudel** [comment]
xD

**Pull request merged**